.github/workflows/golangci-lint.yml

```diff
@@ -13,11 +13,11 @@ jobs:
     steps:
       - uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
         with:
-          go-version: 1.19
+          go-version: 1.21
```

Comment:
I think this should be bumped to 1.20? Also, this and the above change to go_test.yml should be in a separate PR.

Reply:
I'll move them to another PR no problem, but they're in here because the workflows failed, probably mainly because of `errors.Join`.

Reply:
Done in #38